website/blog/2024-08-20-introducing-fe-libs-tailwind.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Introducing - Frontend Libs Tailwind
+description: Explaining the idea behind switching to Tailwind, our first impressions and some examples.
+slug: introducing-fe-libs-tailwind
+authors: obradovic
+date: 2024-08-20
+tags: [eightshift, tailwind]
+hide_table_of_contents: false
+---
+
+It’s been quite a lot of changes with Eightshift DevKit over the last few months, but switching to Tailwind for our Frontend Libs is by far the most significant one.
+<!--truncate-->
+
+## Why Tailwind?
+
+We wanted to speed up the development time and make the projects easier to maintain. Another reason is to make the projects lighter and faster. Our previous setup was quite powerful, but there were a few projects where we almost reached the limit of the system.
+
+We decided to create a version of Frontend Libs that uses Tailwind instead of SCSS and to try it out on a few of our new projects and then decide whether to continue using it or to go back to standard Frontend Libs for our default setup. What were the benefits and our initial impressions?
+
+## Benefits of the new setup
+
+Here are the stats after testing the setup:
+
+- sites are **5x** times faster on the frontend and **3x** times faster on the backend.
+- load time is absolutely **minimal**
+- **50%** fewer DOM elements
+- build time is **10x+** times faster.
+- final bundle sizes for JS and CSS are **3x** smaller.
+
+Additional benefit is that there are a lot of Tailwind community sites with ready-made components, which allow you to copy the component and modify it.
```

Comment:
```suggestion
Also, since Tailwind is so popular, there are many sites with ready-made components that developers and designers can use as a base.
```